costmodel/costmodel.go

```diff
@@ -2418,17 +2409,17 @@ func QueryRange(cli prometheusClient.Client, query string, start, end time.Time,
 		return nil, err
 	}
 
-	_, body, warnings, err := cli.Do(context.Background(), req)
+	resp, body, warnings, err := cli.Do(context.Background(), req)
 	for _, w := range warnings {
-		klog.V(3).Infof("%s", w)
+		klog.V(3).Infof("Warning '%s' fetching query '%s'", w, query)
```

Comment:
Just a thought: for consistency, should we try to stick to the `[Error]`, `[Warning]`, `[Profile]` kind of log prefixing?